changelog/fragments/better-cleanup.yaml

```diff
@@ -0,0 +1,4 @@
+entries:
```

Comment:
Since this command was added into master only, I don't think a CL fragment is necessary.